stable-diffusion/stable-diffusion.ipynb

```diff
@@ -0,0 +1,437 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0c10ed41-57e2-4e7f-a34d-16c94dccbb67",
+   "metadata": {},
+   "source": [
+    "# GUI for Stable Diffusion\n",
+    "\n",
+    "# Stable Diffusion\n",
+    "Written by Jasmine Sandhu (Acknowledgements: Jim Bednar, Maxime Liquet, Philipp Rudiger)<br>\n",
+    "Created: Jan, 2023<br>\n",
+    "Last updated: Jan, 2023"
```

Comment:
You no longer need to include this metadata header in the notebook, this will be included automatically during the doc build, pulling information from the `anaconda-project.yml` file.